### DISCUSSION TITLE
Conditionnal value reset of Options field

### DISCUSSION THREAD

**Discussion by diogenesbrussels** [Budibase/budibase#6410] [Help] [answered]
Hello,

I have a question about a problem of value change according to conditions.

Here is an example: I have two Options fields (displayed with Radio Buttons) in an update form.
If the selection of field 1 is A, B, C, D the field 2 should be displayed with the value from the database and can be altered.
If the selection of field 1 is E, F, G field 2 should be hidden and the value contained should be reset to zero (NULL / no value selected).
With the addition of conditions I can hide field 2 but I cannot delete/reset the value in field 2.

I don't see a 'Current value' parameter in the Update setting list of Conditions.
Is there a way to do this? This happens many times in my forms.

Thank you very much!

**Comment by melohagan** [comment, marked as answer]
Hi @diogenesbrussels 

Could you make use of the **Update Field Value** action? https://docs.budibase.com/docs/actions#update-field-value
You can set the *Type* to reset.

**Reply by diogenesbrussels** [reply]
Hi @melohagan

I got a result with this solution, thank you! 
I would have preferred to be able to group the different conditions in the same place for future maintenance but that's a detail.
